packages/core/src/tag/tag.element.scss

```diff
@@ -27,103 +22,36 @@
   color: var(--color);
   font-size: var(--font-size);
   padding: var(--padding);
-  height: var(--size);
   white-space: nowrap;
-
-  .tag-icon,
-  .tag-badge,
-  .tag-content,
-  .tag-close-icon {
-    @include vertical-center-content;
-    height: calc(var(--size) - #{$cds-token-space-size-2});
-  }
-
-  .tag-badge {
-    height: calc(var(--size) - #{$cds-token-space-size-1});
-  }
-
-  ::slotted(cds-badge) {
-    transform: translateY(calc(-1 - #{$cds-token-space-size-2}));
-    --size: #{$cds-token-space-size-7};
-  }
-
-  .tag-badge {
-    transform: translateY(calc(-1 * #{$cds-token-space-size-1}));
-
-    // using empty containers here b/c firefox doesn't align text-only
-    // tags correctly alongside tags with containers in them
-    &.empty {
-      visibility: hidden;
-      width: #{$cds-token-space-size-1};
-    }
-  }
-
-  .tag-content {
-    padding: 0 #{$cds-token-space-size-4};
-  }
-
-  .tag-icon {
-    overflow: hidden;
-    width: calc(var(--size) - (2 * var(--border-width) + (2 * #{$cds-token-space-size-1})));
-
-    // using empty containers here b/c firefox doesn't align text-only
-    // tags correctly alongside tags with containers in them
-    &.empty {
-      visibility: hidden;
-      width: #{$cds-token-space-size-2};
-    }
-  }
-
-  .tag-close-icon {
-    overflow: hidden;
-    width: calc(var(--size) - #{$cds-token-space-size-3});
-
-    cds-icon {
-      cursor: pointer;
-    }
-  }
-
-  .tag-icon + .tag-content {
-    padding-left: calc(#{$cds-token-space-size-4} / 2);
-  }
-}
```

Comment:
The lhe attr was able to clean up the line height/spacing so the spacing is generic and scales evenly when the theme changes. This seems to work well for all browsers.